sharktank_models/action.yml

```diff
@@ -19,21 +19,39 @@ inputs:
     description: "Type of target to test"
   GPU:
     description: "Type of GPU to test"
+  BACKEND:
+    description: "Type of backend to test"
   EXTERNAL_TEST_FILE_DIRECTORY:
     description: "The directory of test files and configurations"
   QUALITY_TEST_DIRECTORY:
     description: "The directory of quality tests"
   BENCHMARK_TEST_DIRECTORY:
     description: "The directory of benchmark tests"
+  IREE_TEST_FILES:
+    description: "Directory of the iree test files"
```

Comment:
There are enough variables in this file now that they should have documentation. Can you add a comment to the top of this file, or inline, explaining what is expected from each variable?

For example, provide sample values for `TARGET`, `GPU`, and `BACKEND` (these are overlapping, so be explicit here to show how they complement one another). Then for these directory variables, explain what files are expected to exist at each path, if the paths can be omitted to skip those types of tests, etc.

We may also want to instead provide all of these through a configuration file instead of individual variables, as is already done for the other test suites.